src/router/index.ts

```diff
@@ -20,6 +20,12 @@ export const routes: Array<RouteRecordRaw> = [
     component: () =>
       import(/* webpackChunkName: "generic" */ "@/views/GenericBadges.vue"),
   },
+  {
+    path: "/dynamic",
+    name: "Dynamic",
+    component: () =>
+      import(/* webpackChunkName: "generic" */ "@/views/DynamicBadges.vue"),
```

Comment:
Chunk names must be unique.
```suggestion
      import(/* webpackChunkName: "dynamicData" */ "@/views/DynamicBadges.vue"),
```